web-console/src/lib/components/connectors/dialogs/elements/PresetServiceElement.tsx

```diff
@@ -0,0 +1,34 @@
+import { ServiceType } from '$lib/types/xgressServices'
+import { AutocompleteElement } from 'react-hook-form-mui'
+import { usePipelineManagerQuery } from 'src/lib/compositions/usePipelineManagerQuery'
+
+import { useQuery } from '@tanstack/react-query'
+
+export const PresetServiceElement = (props: {
+  serviceType: ServiceType
+  disabled?: boolean
+  onChange?: (value: string | null) => void
+  parentName: string
+}) => {
+  const PipelineManagerQuery = usePipelineManagerQuery()
+  const { data } = useQuery(PipelineManagerQuery.listServices(null, null, props.serviceType))
+  const services = data ?? []
+  return (
+    <>
+      <AutocompleteElement
+        name={props.parentName + '.preset_service'}
+        label={
+          services.length
+            ? 'Optional: connect to a known service'
+            : 'No known services, enter full service configuration'
```

Comment:
```suggestion
            : 'No existing services, add a service or enter full connector details.'
```

Reply:
@lalithsuresh , do we want to replace "known" with "existing"? There was a point earlier how we want to clearly communicate that Feldera doesn't manage services, it can only "register" them